Tests/XCUITests/DownloadFilesTests.swift

```diff
@@ -83,26 +97,32 @@ class DownloadFilesTests: BaseTestCase {
         XCTAssertTrue(app.tables.cells.staticTexts[testBLOBFileSize].exists)
     }
 
-    func testDeleteDownloadedFile() {
+    func testDeleteDownloadedFile() throws {
         downloadFile(fileName: testFileName, numberOfDownlowds: 1)
         navigator.goto(BrowserTabMenu)
         navigator.goto(LibraryPanel_Downloads)
         waitForExistence(app.tables["DownloadsTable"])
-
-        deleteItem(itemName: testFileNameDownloadPanel)
-        waitForNoExistence(app.tables.cells.staticTexts[testFileNameDownloadPanel])
-
-        // After removing the number of items should be 0
-        checkTheNumberOfDownloadedItems(items: 0)
+        if processIsTranslatedStr() == m1Rosetta {
```

Comment:
Rather than skipping the whole test, I decided to let to test to execute until here because the `tearDown()` require the test to be on the `LibraryPanel_Downloads` screen.